src/app/zap-templates/zcl/data-model/chip/group-key-mgmt-cluster.xml

```diff
@@ -31,12 +31,14 @@ limitations under the License.
   </struct>
 
   <enum name="GroupKeySecurityPolicyEnum" type="enum8">
+    <cluster code="0x0752"/> <!-- Joint Fabric Datastore -->
     <cluster code="0x003F"/>
     <item name="TrustFirst" value="0x00"/>
     <item name="CacheAndSync" value="0x01"/>
   </enum>
 
   <struct name="GroupKeySetStruct">
+    <cluster code="0x0752"/> <!-- Joint Fabric Datastore -->
```

Comment:
As above.

Reply:
Copied the enum definition to the JF datastore cluster and submitted the update in the most recent change set.